tests/suites/data-model.js

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright 2024 Digital Bazaar, Inc.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import {createInitialVc, endpointCheck} from '../helpers.js';
+import {
+  assertions,
+} from 'data-integrity-test-suite-assertion';
+import {didResolver} from '../didResolver.js';
+import {expect} from 'chai';
+
+export function dataModelSuite({
+  issuers,
+  suiteName,
+  keyType,
+  vcVersion,
+  credential,
+  mandatoryPointers
+}) {
+  return describe(`${suiteName} - Data Model - VC ${vcVersion}`, function() {
+    this.matrix = true;
+    this.report = true;
+    this.implemented = [...issuers];
+    this.rowLabel = 'Test Name';
+    this.columnLabel = 'Implementation';
+    for(const [name, {endpoints}] of issuers) {
+      const [issuer] = endpoints;
+      // does the endpoint support this test?
+      if(!endpointCheck({endpoint: issuer, keyType, vcVersion})) {
+        continue;
+      }
+      describe(`${name}: ${keyType}`, function() {
+        let securedCredential = null;
+        let proofs = [];
+        before(async function() {
+          securedCredential = await createInitialVc({
+            issuer,
+            vcVersion,
+            vc: credential,
+            mandatoryPointers
+          });
+          if(securedCredential) {
+            proofs = Array.isArray(securedCredential.proof) ?
+              securedCredential?.proof : [securedCredential?.proof];
+            // only test proofs that match the relevant cryptosuite
+            proofs = proofs.filter(p => p?.cryptosuite === suiteName);
+          }
+        });
+        beforeEach(function() {
+          this.currentTest.cell = {
+            rowId: this.currentTest.title,
+            columnId: this.currentTest.parent.title
+          };
+        });
+        function assertBefore() {
+          expect(
+            securedCredential,
+            `Expected issuer ${name}: ${keyType} to issue a VC`
+          ).to.exist;
+          expect(
+            securedCredential,
+            'Expected VC to be an object'
+          ).to.be.an('object');
+        }
+        it('The publicKeyMultibase value of the verification method MUST ' +
+          'start with the base-58-btc prefix (z), as defined in the ' +
+          'Multibase section of Controller Documents 1.0. A ',
```

Comment:
I'm not sure what *should* be written here. Maybe the ` A ` should just be dropped? Maybe `1.0.A` was meant? For certain, the sentence should end with a `.`.
```suggestion
          'start with the base-58-btc prefix ("z"), as defined in the ' +
          'Multibase section of Controller Documents 1.0.A.',
```

Reply:
@TallTed this looks like a typo resulting my cut and paste job. The line reads: "The publicKeyMultibase value of the verification method MUST start with the base-58-btc prefix (z), as defined in the [Multibase section](https://www.w3.org/TR/controller-document/#multibase-0) of [Controller Documents 1.0](https://www.w3.org/TR/controller-document/)" The A belongs to the next sentence: "A Multibase-encoded ECDSA 256-bit public key value or an ECDSA 384-bit public key value follows, as defined in the [Multikey section](https://www.w3.org/TR/controller-document/#Multikey) of [Controller Documents 1.0](https://www.w3.org/TR/controller-document/). Any other encoding MUST NOT be allowed."

So the A should be removed.

Reply:
@TallTed addressed here: https://github.com/w3c/vc-di-ecdsa-test-suite/pull/95/commits/df7e5726c08069bda91aab71e0ac19af9b08ce3f